connaisseur/sigstore_validator.py

```diff
@@ -36,19 +36,28 @@ def get_cosign_validated_digests(image: str, pubkey: str):
                     digest = sig_data["Critical"]["Image"].get(
                         "Docker-manifest-digest", ""
                     )
+                    # check whether the digest matches "sha256:<hex representation of a sha256 hash>" or
+                    # just plain "<hex representation of a sha256 hash>"
                     if re.match(r"(sha256:)?[0-9A-Fa-f]{64}", digest) is None:
-                        raise Exception(
-                            "digest does not match expected digest pattern."
+                        logging.info(
```

Comment:
Why do we no longer raise here? If we got to this block, we have a valid json with the structure we expect. If then the digest is wrongly formatted, something is wrong and we should fail or is there a valid reason for the digest to be mis-formatted?

And if there is, then the subject of the PR and commit are rather "make parsing cosign more lenient and error handling more specific" or sth similar